SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -2084,11 +2085,19 @@ public void setUseCaffeineCache(boolean useCaffeineCache) {
     this.useCaffeineCache = useCaffeineCache;
   }
 
-  public int getCaffeineCacheTtl() {
-    return caffeineCacheTtl;
+  public int getDeployCaffeineCacheTtl() {
+    return deployCaffeineCacheTtl;
   }
 
-  public void setCaffeineCacheTtl(int caffeineCacheTtl) {
-    this.caffeineCacheTtl = caffeineCacheTtl;
+  public void setDeployCaffeineCacheTtl(int deployCaffeineCacheTtl) {
```

Comment:
Nit — can we remove `caffeine` from the terminology here? Users don't need to be aware of cache choice/impl, and we could change caches in the future if we wanted to without any awkwardness

Reply:
I've also updated the `useCaffeineCache` to `useZKFastCache`